### DISCUSSION TITLE
Dynamically display or hide panels

### DISCUSSION THREAD

**Discussion by philippexe** [grafana/grafana#67349] [Dashboards, Panels & Visualizations]
It would be quite nice to have the ability to dynamically hide panels based on template variable values.
The usecase would be to make the Dashboard more interactive. The variable would be set by a datalink from another panel visualization. 

Example: You have an overview of server metrics, you can click on one to get more details for CPU or RAM or ..., currently the datalink has to point to a new dashboard page, introducing latency, grafana loading screen and generally a media break. Instead dynamically displaying the relevant visualizations on the same dashboard would be nice.

Related: https://github.com/grafana/grafana/discussions/37848
But now for timeranges but all free controlable variables, including custom variables.

**Comment by torkelo** [comment]
The new dashboard architecture we are working on is designed to enable this

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82744. Feel free to continue the discussion around this there. Thank you!
